docs/routing.md

```diff
@@ -0,0 +1,289 @@
+# Routing
+
+SyliusResourceBundle ships with a custom route loader that can save you some time.
+
+## Generating Generic CRUD Routing
+
+To generate a full CRUD routing, simply configure it in your ``config/routes.yaml``:
+
+```yaml
+    app_book:
+        resource: |
+            alias: app.book
+        type: sylius.resource
+```
+Results in the following routes:
+
+```bash
```

Comment:
This is technically not correct. Only the `php bin/console debug:router` is bash. The rest is only text output where no syntax highlighting should be used.
Or maybe we just use the table function of markdown?